src/components/Timeline/updates/TimelineJourneyMilestone.tsx

```diff
@@ -78,15 +89,36 @@ const TimelineJourneyMilestone: React.FunctionComponent<Props> = ({
   function renderContent() {
     return (
       <Grid alignItems="center" container direction="row" item spacing={2}>
-        {changeType === 'complete' && (
+        {changeToRender === 'complete' && (
           <CheckCircleIcon style={{ color: theme.palette.success.main }} />
         )}
-        {changeType === 'incomplete' && (
+        {changeToRender === 'incomplete' && (
           <CancelIcon style={{ color: theme.palette.warning.main }} />
         )}
         <Typography component={Grid} item variant="h6">
           {update.details.milestone.title}
         </Typography>
+        {changeToRender === 'deadline' && (
+          <FlagIcon
+            style={{
+              color: theme.palette.text.secondary,
+              marginLeft: theme.spacing(2),
+            }}
+          />
+        )}
+        {changeToRender === 'deadline' && (
+          <Typography
+            color="textSecondary"
+            component={Grid}
+            item
+            variant="body2"
+          >
+            now due{' '}
+            <ZetkinRelativeTime
+              datetime={update.details.changes.deadline?.to as string}
+            />
```

Comment:
I didn't realise a milestone could exist without a deadline. If it can then it needs special case formatting

Reply:
I'm guessing that's why you had to cast it `as string` here, because the type of `deadline` is `string | null`. But yeah, it can be `null` and I think a message like "[actor] removed the deadline for a milestone" is the simplest way to handle that.

Reply:
@richardolsson let me know if that null case is a thing, and I'll add UI for it

Reply:
Yes, `deadline` can be `null`, and when changing from `string` to `null` that means the user removed the deadline.

Reply:
I should have thought about this when I did the cast. will get on it 👍